src/App.jsx

```diff
@@ -3,33 +3,35 @@ import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import './App.css'
 
-function App() {
-  const [count, setCount] = useState(0)
+const todoList = [
+    {
+      id:1,
+      title: 'Complete assignment',
+    },
+    {
+      id:2,
+      title: 'Complete assignment',
+    },
+    {
+      id:3,
+      title: 'Complete assignment',
+    }
+];
 
+function App() {
   return (
-    <>
-      <div>
-        <a href="https://vite.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+    <div>
+      <h1>Todo List</h1>
+
+      <ul>{todoList.map(function(item) {
+          return (
+            <li key = {item.id}>{item.title}</li>
```

Comment:
@IrinaKhameeva there should be no space between the `key` and it's value. 
```
key={item.id}
```

Also, to make your code shorter and cleaner, you can consider using the arrow function syntax:
```
todoList.map((item) =>  <li></li>)
```